app/js/controllers/dev.js

```diff
@@ -0,0 +1,21 @@
+import React from 'components/react';
+import ReactDOM from 'components/react-dom';
+import { Controller } from 'components/mvc';
+
+import CameraLatestImageView from 'js/views/dev/camera-latest-image';
+
+export default class DevController extends Controller {
```

Comment:
I understand the point, and basically as a first hacky approach for having a demo for the next 24 hours could be valid. Period.
TLDR: We all agree, this is a temporary solution to show case what we can do with the app. We need more thinking.

As you comment, we need to sit down and think how we want to organise not just the notifications, but the communication box -> user. Simplifying it just to a controller or existing controller definitely won't scale much, but we need as well to define what kind of information we will get.

Right now we can have predefined static strings, already configured in the rules, again doesn't seem to me really powerful. Or better said, we are limiting ourselves a lot :).

We need to sit with @aosmond and @Yoric to define this kind of communication, or future communications :)